test/integration/utils_test.go

```diff
@@ -121,37 +128,71 @@ func updateModelWithBackend(t *testing.T, m *v1.Model, testModelBackend *httptes
 	}, "Set model IP/port annotations to direct requests to testBackend instead of the Pod's (non-existant) IP")
 }
 
-func sendRequests(t *testing.T, wg *sync.WaitGroup, modelName string, n int, expCode int, msg string) {
+func sendRequests(t *testing.T, wg *sync.WaitGroup, modelName string, selectorHeaders []string, n int, expCode int, expBodyContains string, msg string) {
```

Comment:
nit: when you move `selectorHeaders` to the end with variadic parameter type, you can avoid passing nil

```suggestion
func sendRequests(t *testing.T, wg *sync.WaitGroup, modelName string, n int, expCode int, expBodyContains string, msg string, selectorHeaders ...string) {
```

Reply:
I think passing nil and using `[]string` is easier to read but that may be because I'm not as experienced with Golang. I think both forms are equally good in this case.

Reply:
leaving